packages/now-cli/test/dev/integration.js

```diff
@@ -220,10 +226,37 @@ async function testFixture(directory, opts = {}, args = []) {
   };
 }
 
-function testFixtureStdio(directory, fn) {
+function testFixtureStdio(
+  directory,
+  fn,
+  { expectedCode = 0, skipDeploy } = {}
+) {
   return async t => {
-    let dev;
     const dir = fixture(directory);
+    const token = await fetchTokenWithRetry();
+    let deploymentUrl;
+
+    // Deploy fixture and link project
+    if (!skipDeploy) {
+      const gitignore = join(fixtureAbsolute(directory), '.gitignore');
+      const gitignoreExists = await fs.exists(gitignore);
+      let { stdout, stderr, exitCode } = await execa(
+        binaryPath,
+        [dir, '-t', token, '--confirm', '--public', '--debug'],
+        { reject: false }
+      );
+      console.log({ stdout, stderr, exitCode });
```

Comment:
Remove?

```suggestion
```

Reply:
If a test is incorrect it could fail to deploy so its good to see why, this happens before now dev runs so I'm going to leave it for now and we can remove if it seems to verbose in the future.